schema/validator.go

```diff
@@ -54,6 +55,28 @@ func (v Validator) Validate(src io.Reader) error {
 			"schema %s: unable to validate", v)
 	}
 
+	if v == MediaTypeManifest {
```

Comment:
Yes. So far I find that schema/*.json should not modified because they only present the generally patten and can't cover all constraints. I'd work out on image code layer.